packages/perseus-editor/src/components/start-coords-quadratic.tsx

```diff
@@ -0,0 +1,95 @@
+import {View} from "@khanacademy/wonder-blocks-core";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import {color, font, spacing} from "@khanacademy/wonder-blocks-tokens";
+import {
+    BodyMonospace,
+    LabelLarge,
+    LabelMedium,
+} from "@khanacademy/wonder-blocks-typography";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+import CoordinatePairInput from "./coordinate-pair-input";
+import {getQuadraticEquation} from "./util";
+
+import type {Coord, PerseusGraphType} from "@khanacademy/perseus";
+
+type Props = {
+    startCoords: [Coord, Coord, Coord];
+    onChange: (startCoords: PerseusGraphType["startCoords"]) => void;
+};
+
+const StartCoordsQuadratic = (props: Props) => {
+    const {startCoords, onChange} = props;
+
+    return (
+        <>
+            {/* Current equation */}
+            <View style={styles.equationSection}>
+                <LabelMedium>Starting equation:</LabelMedium>
+                <BodyMonospace style={styles.equationBody}>
+                    {getQuadraticEquation(startCoords)}
+                </BodyMonospace>
+            </View>
+
+            {/* Points UI */}
+            <View style={styles.tile}>
+                <LabelLarge>Point 1:</LabelLarge>
+                <Strut size={spacing.small_12} />
+                <CoordinatePairInput
+                    coord={startCoords[0]}
+                    labels={["x", "y"]}
+                    onChange={(value) =>
+                        onChange([value, startCoords[1], startCoords[2]])
+                    }
+                />
+            </View>
+            <View style={styles.tile}>
+                <LabelLarge>Point 2:</LabelLarge>
+                <Strut size={spacing.small_12} />
+                <CoordinatePairInput
+                    coord={startCoords[1]}
+                    labels={["x", "y"]}
+                    onChange={(value) =>
+                        onChange([startCoords[0], value, startCoords[2]])
+                    }
+                />
+            </View>
+            <View style={styles.tile}>
+                <LabelLarge>Point 3:</LabelLarge>
+                <Strut size={spacing.small_12} />
+                <CoordinatePairInput
+                    coord={startCoords[2]}
+                    labels={["x", "y"]}
+                    onChange={(value) =>
+                        onChange([startCoords[0], startCoords[1], value])
+                    }
+                />
+            </View>
+        </>
+    );
+};
+
+const styles = StyleSheet.create({
```

Comment:
NIT: Grrr. Same styling as all the [other "start-coords-*" implementations](https://github.com/Khan/perseus/pull/1468/files#diff-c2eba6387e6940c42713ad16223d5c2689f99010e62db4d6e34b2f1dc822c692R58-R78).
No need to change anything in this PR.

Reply:
Ah, I did admittedly copy/paste the entire sinusoid file and build off of it. I should probably break out `Tile` into a separate component. Maybe in a different PR.